### PR TITLE
CI badges

### PR DESCRIPTION
**Pull request by itrushkin**
## New
- Ubuntu test sequence
- Windows test sequence
- 2 badges in the README.md

The test sequence for both operating systems is following:
- Linter (removed on Windows because of https://github.com/savoirfairelinux/flake8-copyright/pull/16)
- PyTest
- Dockerization (removed on Windows because of inability to switch to Windows containers inside Github Actions).
- Interactive API (UNet Kvasir example)
- CLI (Keras CNN MNIST template)

### CODE REVIEW

**Comment by itrushkin** [comment]
Badges could not be rendered. According to [Github documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), scheduled workflows will trigger only after workflow files appear in the default (develop) branch. In order for badges to render, this PR should be merged. Both workflows have been tested locally.

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: Ubuntu (latest)
+
+on:
+  push:
+    branches:
+      - ci_badges
```
Comment (psfoley) [inline .github/workflows/ubuntu.yml]: `ci_badges` is the name of this branch, which I would expect would be deleted after merge. Should the condition to run be pushing to `develop` instead?

Comment (itrushkin) [reply .github/workflows/ubuntu.yml]: Yes, this is for debugging purpose. I am checking the pipeline on GitHub Actions.

**Comment by itrushkin** [comment]
Badges in this branch have been rendered. Resetting the trigger to scheduled.

**Review by psfoley — APPROVED** [review]
LGTM!

**Pull request merged**